src/jshint.js

```diff
@@ -4189,8 +4189,8 @@ var JSHINT = (function() {
       if (state.tokens.next.id === "var") {
         advance("var");
         state.tokens.curr.fud({ prefix: true });
-      } else if (state.tokens.next.id === "let") {
-        advance("let");
+      } else if (state.tokens.next.id === "let" || state.tokens.next.id === "const") {
+        advance(state.tokens.next.id);
```

Comment:
other linting rules: initializers are illegal in es6 here, as are multiple declarations